area/guadosalam.py

```diff
@@ -5,123 +5,71 @@
 import pathing
 import vars
 import xbox
-from paths import GuadoSkip, GuadoStoryline
+from paths import GuadoStart, GuadoSkip, GuadoStoryline
 
 logger = logging.getLogger(__name__)
 game_vars = vars.vars_handle()
 
 FFXC = xbox.controller_handle()
 
 
-def arrival():
+def arrival(checkpoint=0):
```

Comment:
It looks like checkpoint is always `0` when we call this. I don't see us calling this with a different `checkpoint` value. As such, we can probably do:

```python
def arrival():
    checkpoint = 0
    ...
```

It's possible this was added for loading an arbitrary save. If that's the case, that's fine, I'd still like to see this change until we actually hook in the change for loading a save.

Reply:
You've got it exactly right. Checkpoint can either be a static or passed value, and I've used that for save-loading features in the past. I only made it a passed value here because it was the same in the function I referenced when adding this one. It literally makes no difference functionally at this time. :D

Reply:
Adding - I'll go with your recommendation for this instance. If we find an instance in the future, we can circle back.